qemu/tests/cfg/virtio_fs_share_data_multi_backend.cfg

```diff
@@ -69,6 +69,20 @@
                     export_dir_fs2 = /mnt/virtio_fs2_test_nfs
                     nfs_mount_src_fs1 = 127.0.0.1:/mnt/virtio_fs1_test_nfs
                     nfs_mount_src_fs2 = 127.0.0.1:/mnt/virtio_fs2_test_nfs
+        - with_ext_fs:
+            setup_filesystem_on_host = yes
+            dd_of_on_host = '/home/extdisk_virtiofs_test'
+            cmd_losetup_query_on_host = 'losetup -f'
+            cmd_losetup_on_host = 'losetup -f %s'
+            cmd_losetup_d_on_host = 'losetup -d %s'
+            cmd_mkfs_on_host = 'mkfs.%s '
+            cmd_dd_on_host = 'dd if=/dev/urandom of=%s bs=1M count=1024 oflag=direct'
+            variants:
+                - ext3:
+                    fs_on_host = ext3
+                - ext4:
+                    fs_on_host = ext4
+
```

Comment:
Hi Houqi, sorry for the delay.
As the variant is a little big, if add this variant, our cases will be much more, I suggest to take it out as a single one.

Reply:
> Hi Houqi, sorry for the delay. As the variant is a little big, if add this variant, our cases will be much more, I suggest to take it out as a single one.

Hi, @nickzhq and @xiagao 
Does this scenario relate with the multi-backend of virtio_fs?
I think that we can keep it in this cfg if it does,  if not, might we consider creating a new independent cfg file for this case.
Anyway, it depends on how to design the manual case.
CC @zhenyzha

Reply:
Make sense, it is a scenario of a multi-backend.
So, houqi, could you help add a limitation to with_stdev_check test case that it only run on with_nfs_source backend?

Reply:
@nickzhq could you update last comment? thanks.

Reply:
> Make sense, it is a scenario of a multi-backend. So, houqi, could you help add a limitation to with_stdev_check test case that it only run on with_nfs_source backend?

Hello xiaoling, basically, the one of rules we follow is one case/bug on one pr. If you hope me to change code for the limitation to with_stdev_check, I can submit another pr after that this pr is merged because the limitation to with_stdev_check is NOT connected with this pr. Thanks!

Reply:
hi houqi, xiaoling meant that when we add this new variant 'with_ext_fs', more variant combinations will be generated if there is no 'filter', e.g. with_stdev_check.with_ext_fs.xxx, but for with_stdev_check, the only supported combination is 'with_nfs_source', so we have to add a filter like only with_nfs_source to avoid that. 
One more thing, we may also need to confirm with xiaoling if  extra_parameters is a supported combination with with_ext_fs. @nickzhq @xiagao

Reply:
Yes, extra_parameters is supported with ext_fs. But with_stdev_check is only supported on with_nfs_source.@nickzhq